packages/app/src/components/Interpretations/Interpretations.js

```diff
@@ -25,12 +25,12 @@ export class Interpretations extends Component {
             // the other scenario is when the URL changes because of a click inside the Interpretations component
             if (history.location.pathname !== interpretationUrl) {
                 history.push(interpretationUrl);
-            }
 
-            this.props.acSetUiInterpretation({
-                id: interpretation.id,
-                created: interpretation.created,
-            });
+                this.props.acSetUiInterpretation({
```

Comment:
Why was this moved inside the `if`? Is this a bug fix?

Reply:
not a bug fix, but an optimization. see commit description: https://github.com/dhis2/data-visualizer-app/pull/195/commits/32dfdd713f0ae003d51d3062301695fd81c94824